test/parallel/test-tick-processor.js

```diff
@@ -22,6 +22,10 @@ runTest(/LazyCompile.*\[eval\]:1|.*%  UNKNOWN/,
    f();`);
 if (process.platform === 'win32' ||
     process.platform === 'sunos' ||
+    process.platform === 'aix' ||
+    ((process.platform === 'linux') &&
+     (process.arch === 'ppc64') &&
+     (process.config.variables.node_byteorder === 'big')) ||
```

Comment:
The latter should be a helper.

Please also switch to using `common.isWindows` and common.isAix`. :)